wiki/Client/Beatmap_editor/Timing/en.md

```diff
@@ -1,25 +1,96 @@
-# Timing tab
+---
+tags:
+  - red line
+  - red offset
+  - red timing point
+  - uninherited offset
+  - green line
+  - green offset
+  - green timing point
+  - inherited offset
+  - timing setup
+  - copy timing
+  - paste timing
+---
 
-*See also: [How to time songs](/wiki/Guides/How_to_Time_Songs)*
+# Timing
 
-![Screenshot of the timing tab in the editor](/wiki/shared/timing/Timing_base.jpg)
-
-The **timing tab** of the [beatmap editor](/wiki/Client/Beatmap_editor) lets mappers change the timing settings of a [beatmap](/wiki/Beatmap). Timing is usually the first step in creating a new beatmap.
+*For a timing setup guide, see: [How to time songs](/wiki/Guides/How_to_Time_Songs)*\
+*See also: [Beatmapping/Timing](/wiki/Beatmapping/Timing)*
 
-Timing consists of a song's [offset](/wiki/Offset), [beats per minute (BPM)](/wiki/Music_theory/Tempo), and [meter](https://en.wikipedia.org/wiki/Metre_(music)). Correct timing on beatmaps is necessary for players to maintain [accuracy](/wiki/Gameplay/Accuracy) while playing. Incorrectly timed beatmaps cannot pass through the [beatmap ranking procedure](/wiki/Beatmap_ranking_procedure).
+**Timing** is the [beatmap editor](/wiki/Client/Beatmap_editor)'s tab for changing and setting up timing of a [beatmap](/wiki/Beatmap), which is essential for correct song representation. It contains related settings and helpers, and also a [dedicated window](#timing-setup-panel) for working with multiple [timing points](#timing-points), both for musical structure description and beatmap design purposes.
```

Comment:
```suggestion
**Timing** is the [beatmap editor](/wiki/Client/Beatmap_editor)'s tab for changing and setting up the timing of a [beatmap](/wiki/Beatmap), which is essential for correct song representation. It contains timing related settings and tools, and also a [dedicated window](#timing-setup-panel) for working with multiple [timing points](#timing-points), both for musical structure representation and beatmap design purposes.
```

helpers -> tools seems like a better description here, nothing in the timing tab does things for you and helper seems rly humanising?

Reply:
"helper" doesn't mean anything to me (i've never heard this term used to explain a UI before, if that's what it's trying to do).